README.md

```diff
@@ -75,6 +75,39 @@ You can specify different template extensions by setting the `template_extension
 template_extensions: ['txt', 'lqd']
 ```
 
+#### Custom paths
```

Comment:
As mentioned, I'm not a big fan of the term "permalink", what does that even mean?. It's a custom path that is different from the file path, that's why I called it "custom paths",